blockchain/consensus/deleg_cns/src/consensus.rs

```diff
@@ -136,6 +136,7 @@ impl Consensus for DelegatedConsensus {
     {
         crate::validation::validate_block(&self.chosen_one, state_manager, block)
             .await
+            .map_err(|err| *err)
```

Comment:
How about changing the `Error` type to `Box<DelegatedConsensusError>`?

Reply:
Changed